src/formatPhoneNumber/index.js

```diff
@@ -8,6 +8,6 @@ export default (value) => {
   if(![String, Number].some(type => R.is(type, value) )) {
     throwError("Value passed to formatPhoneNumber must be a string or number");
   }
-  const x = value.toString().replace(/\D/g, '').match(/(\d{0,3})(\d{0,3})(\d{0,4})/);
+  const x = value.toString().replace(/\D/g, '').match(/(\d{0,3})(\d{0,3})(\d{0,4})$/);
   return !x[2] ? x[1] : `(${x[1]}) ${x[2]}${x[3] ? `-${x[3]}` : ''}`;
 };
```

Comment:
Do we care about newlines at the end of files?

Reply:
Yes, made the change. I need to add eslint to this repo